internal/splunk/httprequest.go

```
@@ -29,6 +32,23 @@ func HandleHTTPCode(resp *http.Response) error {
 		resp.StatusCode,
 		http.StatusText(resp.StatusCode))
 
+	// Check if there is any error text returned by Splunk that we can append to the error message
+	if resp.Body != nil {
+		var jsonResponse map[string]any
+		bodyString, _ := io.ReadAll(resp.Body)
```

we probably don't want to ignore the error here either.